aster_test.go

```diff
@@ -9,85 +9,85 @@ import (
 )
 
 func TestGetPackageComment(t *testing.T) {
-  tests := []struct{
-    name string
-    directory string
-    packages []string
-    expectedPackageComment string
-  }{
-    {
-      name: "one package",
-      directory: ".",
-      packages: []string{"aster"},
-      expectedPackageComment: "Package aster provides a high level interface for parsing Go code using the stdlib's\n" +
-        "[`go/ast`](https://pkg.go.dev/go/ast) module.\n",
-    },
-    {
-      name: "multi package",
-      directory: ".",
-      packages: []string{"aster", "aster_test"},
-      expectedPackageComment: "aster:\nPackage aster provides a high level interface for parsing Go code using the stdlib's\n" +
-        "[`go/ast`](https://pkg.go.dev/go/ast) module.\n" +
-        "aster_test:\nPackage aster_test contains all the tests for the `aster` module.\n",
-    },
-  }
+	tests := []struct {
+		name                   string
+		directory              string
+		packages               []string
+		expectedPackageComment string
+	}{
+		{
+			name:      "one package",
+			directory: ".",
+			packages:  []string{"aster"},
+			expectedPackageComment: "Package aster provides a high level interface for parsing Go code using the stdlib's\n" +
+				"[`go/ast`](https://pkg.go.dev/go/ast) module.\n",
+		},
+		{
+			name:      "multi package",
+			directory: ".",
+			packages:  []string{"aster", "aster_test"},
+			expectedPackageComment: "aster:\nPackage aster provides a high level interface for parsing Go code using the stdlib's\n" +
+				"[`go/ast`](https://pkg.go.dev/go/ast) module.\n" +
+				"aster_test:\nPackage aster_test contains all the tests for the `aster` module.\n",
+		},
+	}
 
-  for _, tc := range tests {
-    t.Run(tc.name, func(t *testing.T) {
-      tc := tc
-      t.Parallel()
+	for _, tc := range tests {
+		tc := tc
```

Comment:
☝️ Note that previously the `tc` variable was declared within the `t.Run`, _after_ it's already been used as `t.Run`'s first argument.